### ISSUE TITLE
Missing autofix for methods related to jQuery.position.*

### ISSUE THREAD

**Issue by flovogt** [SAP/ui5-linter#577]
# Missing autofix for methods related to jQuery.position.*

Methods related to `jQuery.position.*` are deprecated. Therefore an autofix should be offered by UI5 linter.

Deprecated APIs:
- [ ] jQuery.position.*

## Deprecated Usage

````js
var scrollbarWidthR = jQuery.position.scrollbarWidth();
````

## Recommended Usage

````js
sap.ui.define([
   "sap/ui/thirdparty/jqueryui/jquery-ui-position"
], (jQuery) => {
  var scrollbarWidthR = jQuery.position.scrollbarWidth();
});
````

**Comment by RandomByte** [comment]
> Will be implemented in CPOUI5FOUNDATION-990.

Note: Since the mentioned API is currently not detected as a problem by UI5 linter, we decided to move the implementation of a detection, plus an eventual autofix, into a dedicated BLI. This should streamline the implementation of 990.

Related issues:

- https://github.com/SAP/ui5-linter/issues/577 (this)
- https://github.com/SAP/ui5-linter/issues/579
- https://github.com/SAP/ui5-linter/issues/580

**Comment by flovogt** [comment]
Will be implemented in CPOUI5FOUNDATION-1048.
